x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlFeatures.java

```diff
@@ -12,10 +12,19 @@
 import org.elasticsearch.features.NodeFeature;
 
 import java.util.Map;
+import java.util.Set;
 
 public class EsqlFeatures implements FeatureSpecification {
+
+    public static final NodeFeature ASYNC_API_SUPPORTED = new NodeFeature("esql.async_api_supported");
```

Comment:
could we please call this "ASYNC_QUERY_API.xx" ?

Reply:
No problem with changing it to anything you like but... what should be "xx"?
Some area of the API/some action?

Reply:
@ChrisHegarty gentle reminder to give your feedback here :)

Reply:
`ASYNC_QUERY` is a good name.